plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixClientModule.java

```diff
@@ -97,6 +99,9 @@ protected void setup(Binder binder)
         configBinder(binder).bindConfig(JdbcMetadataConfig.class);
         configBinder(binder).bindConfig(JdbcWriteConfig.class);
 
+        configBinder(binder).bindConfig(DynamicFilteringJdbcConfig.class);
```

Comment:
I'm guessing this is needed because phoenix doesn't derive from JdbcPlugin.
Changes for Phoenix connector should go into separate commit and need to be explicitly covered by query runner tests.